### PR TITLE
git hacking

### PR DESCRIPTION
**Pull request by quvide**

### CODE REVIEW

File: mysite/polls/tests.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+from django.urls import reverse
+
+from .models import Question
+
+pytestmark = pytest.mark.django_db
+
+
+def create_question(question_text, days):
+    # helper function
+    time = timezone.now() + datetime.timedelta(days=days)
```
Comment (juyrjola) [inline mysite/polls/tests.py]: Use freezegun to freeze time in order to make tests deterministic

File: mysite/polls/tests.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    def test_no_questions(self, client):
+        """Ensure empty index page is displayed correctly."""
+        response = client.get(reverse("polls:index"))
+        assert response.status_code == 200
+        assert "No polls are available." in str(response.render().content)
+        assert len(response.context["latest_question_list"]) == 0
+
+    def test_past_question(self, client):
+        """Ensure normal questions show."""
+        create_question(question_text="Past question", days=-30)
```
Comment (juyrjola) [inline mysite/polls/tests.py]:
```python
question = create_question(question_text="Past question", days=-30)
...
assert response.context["latest_question_list"] == [question]
```

**Comment by quvide** [comment]
Issues should be fixed now @juyrjola

**Comment by juyrjola** [comment]
Other than that one issue, LGTM!

**Pull request closed without merging**